src/components/facets/components/CurrentRefinement.jsx

```diff
@@ -5,6 +5,7 @@ import { useRecoilValue } from 'recoil'
 // import config file for state of facets
 import { currencySymbolAtom } from '@/config/currencyConfig'
 import { hitsConfig } from '@/config/hitsConfig'
+import React from 'react'
```

Comment:
Might be more efficient just to import { Fragment } from react

Reply:
Thanks Matt it's changed ! Merging now